examples/complete/main.tf

```diff
@@ -2,11 +2,38 @@ provider "aws" {
   region = var.region
 }
 
+data "aws_caller_identity" "current" {}
+
+locals {
+  sqs_queue_arn_prefix = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}"
+}
+
+resource "aws_sqs_queue" "this" {
+  count      = 2
+  name       = "${module.this.id}-${count.index}.fifo"
+  fifo_queue = true
+}
+
 module "sns" {
   source = "../../"
 
   allowed_aws_services_for_sns_published = var.allowed_aws_services_for_sns_published
 
+  subscribers = [
```

Comment:
All of this already seems possible without changing the interface of this input, no?

Reply:
agreed, reverted these changes